Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2014,6 +2014,37 @@ - (BOOL)_updateScrollViewContentInsetsIfNecessary
 #endif
 }
 
+- (void)_beginAutomaticLiveResizeIfNeeded
+{
+    if (_liveResizeParameters)
+        return;
+
+    if (!self.window)
+        return;
+
+    // FIXME: This should use explicit live resize notifications instead of inferring it,
+    // especially since this means we may do some spurious live resizes.
+    if (self.window.windowScene.activationState != UISceneActivationStateForegroundInactive)
+        return;
+
+    [self _beginLiveResize];
+    
+    _endLiveResizeNotificationObserver = [[NSNotificationCenter defaultCenter] addObserverForName:UISceneDidActivateNotification object:self.window.windowScene queue:NSOperationQueue.mainQueue usingBlock:makeBlockPtr([weakSelf = WeakObjCPtr<WKWebView>(self)] (NSNotification *note) {
```

Comment:
Nit - you could probably omit the `note` parameter too, since it's unused in this block.